docs/content/dev/getting-started.md

```diff
@@ -1,60 +1,125 @@
 # Getting started
 
-This document will help you with setting up the development environment for HedgeDoc 2.
-
-## Preparation
-
-### Preparing the directory
+To run Hedgedoc 2.0 you need three components: the backend, the frontend and the reverse proxy.
+
+Backend and Frontend are included in Hegdedoc repo. The reverse proxy can be chosen by preference. For development, we
+recommend caddy and the provided configuration.
+
+## Local dev quick guide with real backend
+
+This describes the easiest way to start a local development environment. For other deployments follow the description
+below.
+
+1. Install node (at least Node 14, we recommend Node 18), yarn and caddy
+2. Clone [repo](https://github.com/hedgedoc/hedgedoc.git) and go into directory
+     ```shell
+     git clone https://github.com/hedgedoc/hedgedoc.git
+     cd hedgedoc
+     ```
+3. install dependency in repo root directory (`yarn install`) :hourglass_flowing_sand:
+4. Go to `hedgedoc/commons` directory `cd ../commons`
+5. Build commons (`yarn build`)
+6. Goto `hedgedoc/backend` directory `cd ../backend`
+7. Create `.env` by `cp .env.example .env`
+8. Set session secret by `openssl rand -hex 16 | sed -E 's/(.*)/HD_SESSION_SECRET=\1/' >> .env`
```

Comment:
> In dev mode you don't need a secure secret. So use any value.

So why do we use the complicated method here?

Reply:
fixed